Ost-38

gapoj opened the pull request:
exporting all the entries instead of only the not synched ones
https://dotsur.atlassian.net/browse/OST-38

guillerappi committed:
Update OSTReviewSubmitViewController.m

ost-38 finished

guillerappi committed:
fixed enteredtime on export

guillerappi committed:
Update project.pbxproj

bump version number

mrozanski commented:
@gapoj I'd like to merge this but I'm not sure if the changes to the project file should be included.
Why change the provisioning style to automatic?

gapoj commented:
> @gapoj I'd like to merge this but I'm not sure if the changes to the project file should be included.
> Why change the provisioning style to automatic?

Original it was automatic, I changed it in my previous commit because I had problems to do the code sign in so I changed it back in my last commit that include only that and the version bumped. Let me know what you prefer, is only to the AppStore connect
